BUILD.bazel

```diff
@@ -115,6 +115,12 @@ java_binary(
     runtime_deps = [":gapic_generator_java"] + MAIN_DEPS,
 )
 
+java_binary(
+    name = "protoc-gen-java_gapic_spring",
+    main_class = "com.google.api.generator.spring.Main",
+    runtime_deps = [":gapic_generator_java"] + MAIN_DEPS,
```

Comment:
dropping a note here for adding spring dependencies. Once this pr is in, I'll adjust #1046. this should be the only place in `BUILD.bazel` that needs SPRING_DEPS be added.

Reply:
done in #1046